rmf_task_msgs/msg/Phase.msg

```diff
@@ -0,0 +1,15 @@
+# Name of current phase
+# Title must be kept within the max 12 characters limit
+string title
+
+# Detailed description of the phase - Optional
+string description
+
+# if the duration field below is valid
+bool has_duration
+
+# pending phase: Expected total duration of the phase
+# active phase: Expected remaining time in the phase
+# completed phase: How long the phase actually took
+builtin_interfaces/Duration duration
```

Comment:
As I think about this more, I wonder if we should do the following:

```
# If there was a predicted duration for this phase
bool has_predicted_duration

# pending/completed phase: Original estimation of total phase duration
# active phase: How much time is expected to remain in this phase
builtin_interfaces/Duration predicted_duration

# How much time has actually been spent in this phase
builtin_interfaces/Duration runtime_duration
```

I'd be interested in hearing from the web team (@koonpeng @vallq @ChawinTan) whether that all seems like useful information for the UI, or whether the single context-dependent duration field is enough.